mLRS/CommonTx/jr_pin5_interface.h

```diff
@@ -63,8 +63,8 @@ void (*uart_tc_callback_ptr)(void) = &uart_tc_callback_dummy;
 #define UART_RX_CALLBACK_FULL(c)    (*uart_rx_callback_ptr)(c)
 #define UART_TC_CALLBACK()          (*uart_tc_callback_ptr)()
 
-#ifdef DEVICE_HAS_JRPIN5_FULL_DUPLEX
-#include "jr_pin5_interface_full_duplex.h"
+#ifdef DEVICE_HAS_JRPIN5_NO_TC
```

Comment:
splitting out by esp should do it, IMHO
the h file could then be named also _esp.h